backend/pkg/kafka/deserializer.go

```diff
@@ -200,6 +203,19 @@ func (d *deserializer) deserializePayload(payload []byte, topicName string, reco
 		}, Object: string(payload), RecognizedEncoding: messageEncodingText, Size: len(payload)}
 	}
 
+	// 6. Test for msgp
```

Comment:
Would Step 5 (UTF-8 compatible messagee) potentially return a valid msgpack message? Should it be switched with Step 5?

Reply:
UTF-8 encoded msgpack fails at utf8.Valid check, it is successfully getting msgpack unmarshalled. But frontend code fails with below message when message is not json.
![image](https://user-images.githubusercontent.com/11286580/113472173-e1cc8800-947e-11eb-979e-a6415c532945.png)